### PR TITLE
Adding 'About' option under 'Help' menu

### PR DESCRIPTION
**Pull request by shivaahir158**
> ### Motivate of the pull request
> - [x] To address an existing issue. If so, please provide a link to the issue: <issue id>
> - [ ] Breaking new feature. If so, please describe details in the description part.

> ### Describe the technical details -

Draft PR for 'About' under 'Help' menu, as discussed. 

image

image

> #### What is currently done? (Provide issue link if applicable)
> 
> 
> 
>
> #### What does this pull request change?
> 
> 
> 
> 

> ### Which part of the code base require a change
> 
> - [x] Frontend: <Specify frontend components>main.js
> - [ ] Backend: <Specify backend components>
> - [ ] Library: <Specify the library name, e.g. npm packages>
> - [ ] Plug-in: <Specify the plugin name, e.g. Webpack, jtest>
> - [ ] Documentation
> - [ ] Regression tests
> - [ ] Continuous Integration (CI) scripts

> ### Impact of the pull request

> - [ ] Require a change on Quality of Results (QoR)
> - [ ] Break backward-compatibility. If so, please list who may be influenced.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -218,8 +218,19 @@ const template = [
           await shell.openExternal('https://rapidpowerestimator.readthedocs.io/en/latest/');
         },
       },
+      {
+        label: 'About',
+        click: () => {
+          dialog.showMessageBox(mainWindow, {
+            type: 'info',
+            title: 'Rapid Power Estimator',
+            message: 'Rapid Power Estimator\nVersion: 0.0.1\nAuthor: © 2024 Rapid Silicon Inc. \nType: Engineering',
```
Comment (ravikiranchollangi) [inline main.js:227]: We should not hardcode this, it must be read the from package json file.

Comment (shivaahir158) [reply main.js]: Done, We have RPE name as rapid_power_estimator in package.json, is there a way we can change it to 'Rapid Power Estimator'?

Comment (ravikiranchollangi) [reply main.js]: Read from these keys instead ?

  "productName": "Rapid Power Estimator",
    "copyright": "©2024 Rapid Silicon, Inc.",

File: main.js [hunk 2]
```diff
@@ -218,8 +218,24 @@ const template = [
...
         },
       },
+      {
+        label: 'About',
+        click: () => {
+          // dynamically reading package.json
+          const packagePath = path.join(__dirname, 'package.json');
+          const packageData = JSON.parse(fs.readFileSync(packagePath, 'utf8'));
+      
+          const message = `${packageData.build.productName}\nVersion: ${packageData.version}\nAuthor: ${packageData.author}\nCopyright: ${packageData.build.copyright}\nType: Engineering`;
```
Comment (ravikiranchollangi) [inline main.js:228]: Type is still hardcoded here, please remove it.
Also confirm that production build has same data.

**Pull request merged**